test/helpers/default-functions.ts

```diff
@@ -132,6 +133,7 @@ export function defaultFunctions(): void {
 			await browser.url( `${host}/wiki/${title}?action=edit` );
 
 			// wait for javascript to settle
+			// eslint-disable-next-line wdio/no-pause
```

Comment:
Of course. This task is to do exactly that: https://phabricator.wikimedia.org/T345423. The point here is to suppress excessive warnings which run on every test run and distract from new and otherwise untracked issues.

Reply:
Yes, that makes sense. Thanks for linking the ticket as well.